doc/nrf/libraries/caf/sensor_manager.rst

```diff
@@ -160,6 +165,52 @@ To use the sensor trigger, complete the following steps:
 .. note::
     |only_configured_module_note|
 
+Enabling passive power management
+=================================
+
+The |sensor_manager| can react to ``power_down_event`` and ``wake_up_event``.
+This functionality is called passive power management.
```

Comment:
In this mode sensor would not be able to wake up nor suspend the whole system.

Reply:
Is it really necessary to write it down? There is a sentence that says the module `can react to ...`, is it required to mention what doesn't it do?

Reply:
I just feel that this is not stated clearly enough. Especially that below we have an information about enabling triggers.